code/src/App.js

```diff
@@ -1,9 +1,87 @@
-import React from 'react';
+import React, { useState}  from 'react';
```

Comment:
Nice structure with your components! I like that they comes in the same order as the whole page is built.